lib/src/bazelify/initialize.dart

```diff
@@ -339,9 +339,15 @@ analyzer:
     } else {
       final analysisOptions =
           loadYaml(await analysisOptionsFile.readAsString());
-      final analyzerConfig = analysisOptions['analyzer'];
-      final excludes =
-          analyzerConfig == null ? null : analyzerConfig['exclude'];
+      final analyzerConfig = analysisOptions != null
+          ? analysisOptions['analyzer']
+          : const {
```

Comment:
Running an IDE without those excludes makes it very sluggish while the analyzer does it's thing. We should fail with instructions, or edit the `.analysis_options` file to add the excludes. Otherwise it makes the experience working in the dart + bazel world so bad.